src/tests/functional/shared_test_classes/src/subgraph/memory_LSTMCell.cpp

```diff
@@ -274,8 +280,14 @@ void MemoryLSTMCellTest::create_pure_tensor_iterator_model() {
     auto out_hidden = tensor_iterator->get_iter_value(H_o, -1);
     auto out_cell = tensor_iterator->get_iter_value(C_o, -1);
 
-    out_hidden.get_tensor().set_element_type(element_type);
-    out_cell.get_tensor().set_element_type(element_type);
```

Comment:
@itikhono was `set_element_type()` required? May it be just removed instead of replacing?